### PR TITLE
Adding user facing documentation to README

### PR DESCRIPTION
**Pull request by krmahadevan**
Fixes # .

### Did you remember to?

- [ ] Add test case(s)
- [ ] Update `CHANGES.txt`

### CODE REVIEW

**Comment by krmahadevan** [comment]
ping @cbeust / @juherr

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -21,4 +21,12 @@ Always make sure your issue is happening on the latest TestNG version. Bug repor
...
-This pull request will be a perfect start to find the fix :)
\ No newline at end of file
+This pull request will be a perfect start to find the fix :)
+
+# We encourage pull requests that:
+  
+  * Add new features to TestNG (or)
+  * Fix bugs in TestNG
+  
+  If your pull request involves fixing SonarCube issues then we would suggest that you please discuss this with the
```
Comment (cbeust) [inline .github/CONTRIBUTING.md]: It's spelled "SonarQube".

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -4,3 +4,11 @@ Fixes # .
 
 - [ ] Add test case(s)
 - [ ] Update `CHANGES.txt`
+
+We encourage pull requests that:
+
+* Add new features to TestNG (or)
+* Fix bugs in TestNG
+
+If your pull request involves fixing SonarCube issues then we would suggest that you please discuss this with the
```
Comment (cbeust) [inline .github/PULL_REQUEST_TEMPLATE.md]: Ditto.

**Comment by krmahadevan** [comment]
@cbeust - I fixed the typos.

**Pull request merged**